lisp/mastodon-tl.el

```diff
@@ -160,7 +160,9 @@ also render the html"
    (with-temp-buffer
      (insert (decode-coding-string string 'utf-8))
      (when render
-       (shr-render-region (point-min) (point-max)))
+       (progn
+         (customize-set-value 'shr-use-fonts nil)
```

Comment:
Any reason to use `customize-set-value` over `let`?

```elisp
(when render
  (let ((shr-render-region nil))
    (shr-render-region (point-min) (point-max))))
```

They appear to behave the same when opening up a timeline.